src/runtime/cli/main.go

```diff
@@ -331,7 +333,7 @@ func handleShowConfig(context *cli.Context) {
 			fmt.Fprintf(defaultOutputFile, "%s\n", file)
 		}
 
-		exit(0)
+		exitFunc(0)
```

Comment:
Would prefer just calling os.Exit() here. The indirection to call exitFunc was used for testing. Now that we have got rid of exit_test.go, this should no longer be needed.

Reply:
handleShowConfig() can't call os.Exit() when in unit test.